### DISCUSSION TITLE
toFixed or equiv

### DISCUSSION THREAD

**Discussion by roybarber** [mockoon/mockoon#572] [Help] [answered]
Hey Everyone, is it possible within the current setup to use toFixed or equivelent in the templating?

Heres an example of what im doing:

```
{{setVar 'metersRequired' (queryParam 'sqMeters')}}
{{setVar 'packCoverage' 1.1}}
{{setVar 'packsRequred' (divide metersRequired packCoverage)}}
{{setVar 'packPrice' 42.99}}
{{setVar 'sqmPrice' (divide packPrice packCoverage)}}

{
  "packQty": "{{packsRequred}}",
  "sqmPrice": "{{sqmPrice}}",
  "totalPrice": "{{multiply packPrice packsRequred}}",
  "currancy": "£"
}
```

and the output:

```
{
    "packQty": "6.363636363636363",
    "sqmPrice": "39.08181818181818",
    "totalPrice": "273.57272727272726",
    "currancy": "£"
}
```

I need to round up the output to the nearest whole number for packQty and toFixed(2) for totalPrice & sqmPrice

**Comment by 255kb** [comment, marked as answer]
Hi Roy, to round, you can use the recently added [`floor`](https://mockoon.com/docs/latest/templating/mockoon-helpers/#floor) and [`ceil`](https://mockoon.com/docs/latest/templating/mockoon-helpers/#ceil) helpers. For a `round` helper matching the `Math.round()`, we need to add it. I opened a new issue to track this: https://github.com/mockoon/commons-server/issues/32
There is currently no `toFixed` helper. I also created an issue for this: https://github.com/mockoon/commons-server/issues/31
In the meantime, I can suggest that you use a combination of `split` and `substr` but I guess it's not very practical!
